server/services/v1/sessions.go

```diff
@@ -62,15 +62,7 @@ func (sessMgr *SessionManager) Create(ctx context.Context, reloadVerOutside bool
 	if err != nil {
 		return nil, err
 	}
-	var tenant *metadata.Tenant
-	if namespaceForThisSession == metadata.DefaultNamespaceName {
-		tenant, err = sessMgr.tenantMgr.CreateOrGetTenant(ctx, sessMgr.txMgr, metadata.NewDefaultNamespace())
-	} else {
-		tenant, err = sessMgr.tenantMgr.GetTenant(ctx, namespaceForThisSession, sessMgr.txMgr)
-	}
-	if tenant == nil {
-		return nil, api.Errorf(api.Code_NOT_FOUND, "Tenant %s not found", namespaceForThisSession)
-	}
+	tenant, err := sessMgr.tenantMgr.GetTenant(ctx, namespaceForThisSession, sessMgr.txMgr)
```

Comment:
how will we create a default namespace if it doesn't exist?

Reply:
It's created in the main.